src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -739,6 +740,7 @@ private void ResolveConflictToLocal(ReplicationDocumentsPositions doc, ChangeVec
                         doc.Collection);
                 }
             }
+            Interlocked.Decrement(ref _database.DocumentsStorage._hasConflicts);
```

Comment:
Same note regarding concurrency, and it is probably better to make this a method, rather than expose internal state